### PR TITLE
Add MITNFA, change artistic property to modification-related-obligations, fix existing properties of artistic licenses

### PR DESCRIPTION
**Pull request by willebra**

### CODE REVIEW

File: license-classifications.yml [hunk 1]
```diff
@@ -149,9 +149,14 @@ categories:
...
 
-  - name: "property:artistic-license"
+  - name: "property:modification-related-obligations"
     description: >-
-      This property is included only in the Artistic licenses.
+      This property indicates that the license contains obligations that are triggered by modifying
+      the licensed content. The scope of obligations could vary, but typically it is about somehow
+      making sure that the recipients of the modified content don't attribute the modifications to
+      the wrong author and/or that the unmodified content is available to the recipients. The 
+      obligations go beoynd of mere marking of the modifications. Artistic-type licenses have this
```
Comment (sschuberth) [inline license-classifications.yml:158]: There seem to be trailing spaces here.

File: license-classifications.yml [hunk 2]
```diff
@@ -149,9 +149,14 @@ categories:
...
-  - name: "property:artistic-license"
+  - name: "property:modification-related-obligations"
     description: >-
-      This property is included only in the Artistic licenses.
+      This property indicates that the license contains obligations that are triggered by modifying
+      the licensed content. The scope of obligations could vary, but typically it is about somehow
+      making sure that the recipients of the modified content don't attribute the modifications to
+      the wrong author and/or that the unmodified content is available to the recipients. The
+      obligations go beoynd of mere marking of the modifications. Artistic-type licenses have this
+      property, but also some others.
```
Comment (mmurto) [inline license-classifications.yml:159]: Is the category of Artistic-type licenses large and so property defining, that it should be separately mentioned here, in addition to using it in their classification?

Comment (willebra) [reply license-classifications.yml:159]: We currently have 4 versions of artistic licenses and MITNFA. So a total of 5 instances of this new/changed property. Considering we had a property artistic, this is also reasonably background. Further, artistic lienses are well-known to have this type of properties, whereas MITNFA was totally new to me, at least. So worth spending one line in classification for this.

**Comment by sschuberth** [comment]
Before merging, please rebase this PR onto latest `main` to make the new `REUSE` check work properly.

**Comment by sschuberth** [comment]
> Before merging, please rebase this PR onto latest `main` to make the new `REUSE` check work properly.

@willebra, @mmurto, I've lost a bit track of your discussion. Are we good to merge from your perspective?

**Review by mmurto — APPROVED** [review]

**Pull request merged**